### DISCUSSION TITLE
What can i do with the dish washer switch&temperature?

### DISCUSSION THREAD

**Discussion by drheck** [QuickSander/homebridge-mieleathome#44] [General]
Hi Quicksander,

what can i do with this plugin?

I got my new smart dish washer yesterday and i have installed your plugin.
Now i ask me: what is the functionality?

I would like to power on/off, start/Stop a specific program (with options) and so on.
The API supports all this...

The plugin for siemens homeconnect could do this:
https://github.com/thoukydides/homebridge-homeconnect/blob/master/README.md

So my question ist this planned for the future?

Thank you for an answer.

kind regards,
Peter

**Comment by dkossman** [comment]
same question here.  though i've been unable to log into the Miele iOS app - their service appears to have been broken for the last week or so.
